python/subunit/test_results.py

```diff
@@ -467,6 +469,8 @@ def __init__(self, result, filter_error=False, filter_failure=False,
             parameter for efficiency.
         :param fixup_expected_failures: Set of test ids to consider known
             failing.
+        :param renames: Optional list of regexes to apply to test names
```

Comment:
Actually a list of functions, but probably makes more sense to be a single function, even if the commandline interface is multiple substitution patterns.

Reply:
Done.